test/model-method.js

```diff
@@ -249,6 +258,39 @@ describe('ModelMethod - Loopback 3.0', function() {
             expect(methods).to.have.property('prototype.testMethod');
             expect(methods).to.not.have.property('testMethod');
             expect(methods['prototype.testMethod']).to.not.have.property('isStatic');
+            expect(methods['prototype.testMethod']).to.not.have.property('id');
+            expect(methods['prototype.testMethod']).to.not.have.property('facetName');
+            expect(methods['prototype.testMethod']).to.not.have.property('name');
+            done();
+          });
+        });
+      });
+  });
+
+  it('loading JSON should have correct method name and isStatic flag', function(done) {
+    ModelMethod.create(
+      {
+        modelId: userModel.id,
+        name: 'prototype.testMethod',
```

Comment:
Sorry for repeating myself, but isn't this invalid API usage? IMO, the `name` should be always the short name without `prototype` prefix.